includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -130,6 +139,24 @@ public static function render_vary_header_field() {
 		<?php
 	}
 
+	/**
+	 * Render content negotiation field.
+	 */
+	public static function render_content_negotiation_field() {
+		$value = \get_option( 'activitypub_content_negotiation', '1' );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" id="activitypub_content_negotiation" name="activitypub_content_negotiation" value="1" <?php checked( '1', $value ); ?> />
+				<?php \esc_html_e( 'Differentiates how content is shown to browsers versus Fediverse services.', 'activitypub' ); ?>
+			</label>
+		</p>
+		<p class="description">
+			<?php \esc_html_e( 'Disable this if your site starts displaying raw technical data instead of normal pages, or if ActivityPub connections become unstable. This setting helps ensure your site serves the correct content format to browsers.', 'activitypub' ); ?>
```

Comment:
```suggestion
			<?php \esc_html_e( 'Content negotiation ensures your site displays regular web pages to browsers and machine-readable data to Fediverse services. Disable this if your site shows raw technical data to visitors or if ActivityPub connections have issues.', 'activitypub' ); ?>
```
A bit more detail on what content negotiation is.
"if ActivityPub connections have issues" still feels ambiguous, is there anything more specific we could say there? If posts don't appear on remote Fediverse timelines?